ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -231,12 +272,17 @@ if [[ "${APPLY_ALL_SUBNETS}" == "no" ]]; then
         compute_idx=$(( RANDOM % compute_subnets_len))
       ;;
       *)
-        echo "unsupported value for PICKUP_SUBNETS_IN_ORDER"
+        echo "unsupported value for PICKUP_SUBNETS_ORDER"
         exit 2
       esac
       cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$cp_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("control-plane")) | .name][$idx]]' | yq-go r -P - >>${workdir}/controlPlaneSubnets.yaml
       cat "${vpc_info_file}" | jq -c -r --arg z "${zone}" --argjson idx "$compute_idx" '[[.subnets | sort_by(.created_at) | .[] | select(.zone.name==$z) | select(.name|test("compute")) | .name][$idx]]' | yq-go r -P - >>${workdir}/computerSubnets.yaml
     done
+
+    if  [[ $PICKUP_SUBNETS_ORDER == "ascending" ]]; then
```

Comment:
`ascending` means picking up subnets by creation_date ascending order, it is doing right thing, should not interact with the 2nd page stuff.

Reply:
Now, the ascending picks up the subnets with  creation_date (the latest is the first), To verify the bug, I need to confirm the subnets used not on the first page.

Reply:
Then you need to another option for `PICKUP_SUBNETS_ORDER`, something like `2page`, then in the dedicated case branch to check the subnet meet your testing requirement or not, if not, then exit.